qa/workunits/rbd/copy.sh

```diff
@@ -164,6 +164,7 @@ test_remove() {
     echo "testing remove..."
     remove_images
 
+    echo `rbd remove "NOT_EXIST" > /dev/null 2>&1 || echo $?` | grep 2
```

Comment:
I would simplify this and check that it return non-exit status (not exactly 2). Something like we have for shrink check:
```
rbd resize testimg1 --size=128 && exit 1 || true   # shrink should fail
```